jo-rdan/prop-lite
# chore(validations): property validations

# What does this PR do?
* It creates a validateProperty class
* It creates a createProperty method to validate new property details enterred by the user
* It uses regular expressions for validating data.

#### Description of Task to be completed
* It should validate all the data entered by the user, if they are all valid; it should return a 201 HTTP status code. 

#### How should this be manually tested?
* Start the server by typing **npm start**
* Start POSTMAN and use this url: `localhost:5000/api/v1/property` to create a new property.
* Enter all required details. if they are valid it should return a 201 HTTP status with the created property, else it should return a relevant error message.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167123212](https://www.pivotaltracker.com/story/show/167123212)
